rust/relay/src/server.rs

```diff
@@ -353,6 +351,8 @@ where
         self.data_relayed_counter.add(msg.len() as u64, &[]);
         self.data_relayed += msg.len() as u64;
 
+        tracing::trace!(target: "wire", num_bytes = %msg.len());
```

Comment:
So if the port is invalid and it diverges on line 346, the log isn't written, makes sense